pkg/manager/member/startscript/v2/pd_start_script.go

```diff
@@ -181,6 +187,8 @@ const (
 	pdWaitForDnsIpMatchSubScript = `
 componentDomain=${PD_DOMAIN}
 waitThreshold={{ .PDStartTimeout }}
+initWaitTime={{ .PDInitWaitTime }}
+sleep initWaitTime
```

Comment:
This will cause existing PD to be restarted after we upgraded TiDB Operator if `WaitForDnsNameIpMatch` has been enabled before.

BTW, what is the problem without this `sleep`?

Reply:
@csuzhangxc Thanks for  the reminder, the reason we want  to add this option is: currently we do have  the DNS resolved IP check with current pod ip during pd start script. But we saw an racing condition that the check passed but the pd still hit the wrong ip during it restarts. The reason is we have a set of core  dns pods, maybe some of them got the new record, but some does not. In this case we still hit the wrong IP even after the dns resolved IP check passed. Adding this option, we can force a wait of 2*dns_ttl time before pd start to  avoid such racing condition.